src/dbnode/server/server.go

```diff
@@ -318,6 +321,14 @@ func Run(runOpts RunOptions) {
 		SetForceIndexSummariesMmapMemory(cfg.Filesystem.ForceIndexSummariesMmapMemory).
 		SetForceBloomFilterMmapMemory(cfg.Filesystem.ForceBloomFilterMmapMemory)
 
+	// obtain a lock on `filePathPrefix`
```

Comment:
Is this as high as we can easily move this code? Also, can you add a similar comment to the one you put in this P.R and leave it above this code?

Reply:
Sure, I've now extended the comment. The lock acquisition is also move up, it's almost the first thing done, with only the logger initialisation being before it. Let me know in case that's too early.